### ISSUE TITLE
[finiteReco] Update to a new test file (Trac #1893)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1885]
One of the finiteReco tests uses a test file from IceSim 1.9.2 with the IC9 configuration.  Surely we could come up with something better.

finiteReco is an orphaned project, so this belongs to me and KJ at the moment.  Need to find a good home for this project.

Migrated from https://code.icecube.wisc.edu/ticket/1893
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:55", 
    "description": "One of the finiteReco tests uses a test file from IceSim 1.9.2 with the IC9 configuration.  Surely we could come up with something better.\n\nfiniteReco is an orphaned project, so this belongs to me and KJ at the moment.  Need to find a good home for this project.", 
    "reporter": "olivas", 
    "cc": "kjmeagher", 
    "resolution": "fixed", 
    "_ts": "1550067295757382", 
    "component": "combo reconstruction", 
    "summary": "[finiteReco] Update to a new test file", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-10-13T18:15:45", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170118T19:21:59: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "olivas" to ""

**Comment by icecube-trac** [comment]
Trac update at 20170405T16:08:21: 
* olivas commented:

> Need to find an owner for this project.


* olivas changed milestone from "October 2016" to "July 2017"

**Comment by icecube-trac** [comment]
Trac update at 20170422T05:31:35: kjmeagher changed milestone from "July 2017" to "IceRec 5.2"

**Comment by icecube-trac** [comment]
Trac update at 20170912T10:12:31: 
* kjmeagher commented:

> In changeset 158076:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="158076"
> All this test wanted was a geometery, update to use 2016 GCD file. `corsika.000085.000007.10eventsIC9.i3.gz` can now be safely deleted from TESTDATA. closes #1885
> ```


* kjmeagher changed owner from "" to "kjmeagher"
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:55: 
* olivas changed milestone from "IceRec 5.2" to ""
* olivas commented:

> Milestone IceRec 5.2 deleted
